src/kvaser_can_bridge.cpp

```diff
@@ -48,13 +48,16 @@ void can_read()
 
       if (ret  == ReturnStatuses::OK)
       {
-        // Only publish if msg is not CAN FD.
-        // Also, only publish if msg contains
-        // data, is RTR, or is error frame.
-        if (!msg.flags.fd_msg &&
-            (msg.dlc != 0 ||
-             msg.flags.rtr ||
```

Comment:
What happened to the RTR and error frame conditions?

Reply:
This logic says "If the message is NOT an FD_MSG _or_ it has DLC that is not 0 _or_ it is RTR or error frame. The new logic still allows the RTR and error frames, it just specifically excludes a bunch of the information-only messages.